### PR TITLE
Bundle Update on 2016-08-31

### PR DESCRIPTION
**Pull request by deppbot**
**Updated RubyGems:**

- Updated: [i18n-js](http://github.com/fnando/i18n-js), [3.0.0.rc13...3.0.0.rc14](https://github.com/fnando/i18n-js/compare/v3.0.0.rc13...v3.0.0.rc14) ([CHANGELOG](https://github.com/fnando/i18n-js/blob/master/CHANGELOG.md))
- Updated: [schema_plus_columns](https://github.com/SchemaPlus/schema_plus_columns), [0.1.2...0.1.3](https://github.com/SchemaPlus/schema_plus_columns/compare/v0.1.2...v0.1.3)
- Updated: [simple_form](https://github.com/plataformatec/simple_form), [3.2.1...3.3.1](https://github.com/plataformatec/simple_form/compare/v3.2.1...v3.3.1) ([CHANGELOG](https://github.com/plataformatec/simple_form/blob/master/CHANGELOG.md))
- Updated: [slim-rails](https://github.com/slim-template/slim-rails), [3.1.0...3.1.1](https://github.com/slim-template/slim-rails/compare/v3.1.0...v3.1.1)
- Updated: [terminal-table](https://github.com/tj/terminal-table), [1.6.0...1.7.0](https://github.com/tj/terminal-table/compare/1.6.0...1.7.0) ([CHANGELOG](https://github.com/tj/terminal-table/blob/master/History.rdoc))
- Added: [unicode-display_width](http://github.com/janlelis/unicode-display_width), [1.1.0](https://github.com/janlelis/unicode-display_width/tree/v1.1.0)

--

Brought to you by [deppbot](https://www.deppbot.com) - Automated Security and Dependency Updates
81 Pull Requests sent and 8 hours, 27 minutes, and 20 seconds saved so far.

<sub>Feedback or Bug Reports? Please let us know via [Email](mailto:ask@deppbot.com) or [GitHub Issues](https://github.com/deppbot/feedback).</sub>

### CODE REVIEW

File: Gemfile [hunk 1]
```diff
@@ -177,7 +177,7 @@ gem 'themes_on_rails', '>= 0.3.1', github: 'lowjoel/themes_on_rails',
 
 # Forms made easy for Rails
 gem 'simple_form'
-gem 'simple_form-bootstrap', '>= 1.3.0'
+gem 'simple_form-bootstrap', github: 'allenwq/simple_form-bootstrap'
```
Comment (fonglh) [inline Gemfile]: cannot PR to Joel's repo?

Comment (allenwq) [inline Gemfile]: PR'ed already, waiting for reviewing, we can wait too.

I am also testing if the exceptions are gone with the fix.

Comment (fonglh) [inline Gemfile]: @allenwq joel merged the PR already, so we don't need to use this fork any more?

Comment (allenwq) [inline Gemfile]: New version haven't released yet.

Comment (fonglh) [inline Gemfile]: oh yah I just saw. Poke him?

Comment (allenwq) [inline Gemfile]: Yes, nvm can wait. not that urgent.

**Pull request merged**